packages/react-components/tests/browser/Attachment.spec.tsx

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { expect } from '@playwright/experimental-ct-react';
+import { test as betaTest } from './FlavoredBaseTest';
+import { RichTextSendBox } from '../../src/components/RichTextEditor/RichTextSendBox';
+import { MessageThread, SendBox } from '../../src';
+import { TestMessageThreadWithCustomAttachmentActions } from './TestingComponents/TestMessageThreadWithCustomAttachmentActions';
+
+betaTest.describe('Attachment tests', () => {
+  betaTest.skip(({ isBetaBuild }) => !isBetaBuild, 'The tests should be run for beta flavor only');
+
+  betaTest('Regular SendBox should show attachment progress correctly in group layout', async ({ mount }) => {
+    const component = await mount(
+      <SendBox
+        onSendMessage={async () => {
+          return;
+        }}
+        attachments={[
+          { progress: 0.65, id: 'id1', name: 'test1.pdf' },
```

Comment:
As we are in react-components folder, we can test Attachment cards separately withou SendBox and RichTextSendBox components